extensions/elasticsearch-rest-client-common/deployment/src/main/java/io/quarkus/elasticsearch/restclient/common/deployment/ElasticsearchDevServicesBuildTimeConfig.java

```diff
@@ -54,6 +54,13 @@ public class ElasticsearchDevServicesBuildTimeConfig {
     @ConfigItem(defaultValue = "true")
     public boolean shared;
 
+    /**
+     * Indicates if the Elasticsearch server managed by Quarkus Dev Services
+     * should run in Privileged Mode
+     */
+    @ConfigItem(defaultValue = "true")
+    public boolean privilegedMode;
```

Comment:
I'm not opposed to exposing this setting, but making it default to `true` seems dangerous... especially since this isn't needed when using Docker (as opposed to podman).

But maybe we can avoid this patch altogether by upgrading to ES 8; see [my other comment](https://github.com/quarkusio/quarkus/issues/31960#issuecomment-1475746593). Can you confirm that, please? I personally don't have podman and last time I tried it just didn't work, so...

Reply:
For me there are two options:

- Elasticsearch 8 fixes the issue and we upgrade to it
- It doesn't and we don't expose an option but we enable privileged mode for Podman only (see `ContainerRuntimeUtil#detect...`)